### ISSUE TITLE
6.1.0rc1-1 - When I open a saved session the application freeze

### ISSUE THREAD

**Issue by The-Course** [asbru-cm/asbru-cm#435]
**Describe the bug**
When I open a saved session, the application crashes and I cannot select other sessions or the newly opened shell

**To Reproduce**
ASBRU version 6.1.0rc1-1

1. I open a saved connection/local shell
2. The application seems to be loading and is no longer interactable.

**Screenshots**

![image](https://user-images.githubusercontent.com/46918952/75759789-6a39a000-5d36-11ea-8734-650c7844c310.png)

**Desktop (please complete the following information):**
 - OS: Kubuntu
 - Version 19.10

**Additional context**
Running CLI asbru-cm I find the following output when I try to open a connection :
```
*** unhandled exception in callback:
***   Could not locate symbol vte_terminal_set_bold_is_bright at /opt/asbru/lib/PACTerminal.pm line 456.
***  ignoring at /usr/share/perl5/Gtk3.pm line 572.
```

**Comment by gfrenoy** [comment]
This is also related to a wrong version of libvte.   ```set_bold_is_bright``` is available since 0.52 so I would think that you are using an older version...Let's discuss this in #434.

**Comment by gfrenoy** [comment]
On 19.10, you should not have this message ... so the root reason is #434.  Let me close this for now.
